lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -693,8 +693,8 @@ void SpecialFunctionHandler::handleMakeSymbolic(ExecutionState &state,
     
     const ObjectState *old = it->first.second;
     ExecutionState *s = it->second;
-    
-    if (old->readOnly) {
+
+    if (old->readOnly || isConstant(mo->allocSite)) {
```

Comment:
@delcypher That's the very same good question I asked myself after incurring into this behavior. Apparently `readOnly` is not currently used, and its previous use was to discriminate among "external objects" as implemented in https://github.com/klee/klee/blob/master/lib/Core/Executor.cpp#L536 and subsequent lines. I am thus not sure what's the real meaning of the `readOnly` flag now.